src/auth/auth.constants.ts

```diff
@@ -0,0 +1,8 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
```

Comment:
Use `@nestjs/config` instead of `dotenv`. It's a package from the nest family that is used to load environment variables.

For more information: https://docs.nestjs.com/techniques/configuration